cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericAggregator.java

```diff
@@ -0,0 +1,278 @@
+package com.telefonica.iot.cygnus.aggregation;
```

Comment:
Licence header in every new file, pls.

In addition, a proper line to CHANGES_NEXT_RELEASED regarding the changes in this PR should be added.

Reply:
Fixed in [e26b297](https://github.com/telefonicaid/fiware-cygnus/pull/1798/commits/e26b29747c86742e4397bc57a6ad05d1cfe39ee8)